caddyhttp/httpserver/logger_test.go

```diff
@@ -138,7 +138,10 @@ func TestLoggingToSyslog(t *testing.T) {
 	for i, testCase := range testCases {
 
 		ch := make(chan format.LogParts, 256)
-		server, err := bootServer(testCase.Output, ch)
+		server, err := bootServer(testCase.Output, ch);
+		if err != nil {
+			log.Println("[ERROR] bootServer failed: ", err)
+		}
 		defer server.Kill()
 
 		if err != nil {
```

Comment:
You can remove this error check block because it's redundant to the check in line 142.

Reply:
alright

Reply:
You got it backwards 😅  The error check should be before `defer server.Kill()` because if there's an error, then there is no server to kill.

Reply:
oh no... haha sorry about that. 😅